lib/ok.ex

```diff
@@ -64,9 +64,72 @@ defmodule OK do
   """
   def failure(reason), do: {:error, reason}
 
+  @doc """
```

Comment:
I started this comment line at the line 67, but it pertains to all of the examples that are single line. I should mention that I think these are _great_ examples for a getting started. I think the culmination would be the relatively longer factory example that I mention.